ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -306,6 +306,14 @@ tests:
         IP_STACK=v6
         NETWORK_TYPE=OVNKubernetes
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-ovn-ipv6-required
```

Comment:
There is a e2e-metal-ipi-ovn-ipv6 job defined. Why we need a new one with the same configuration?

Reply:
Looks like one’s optional, and this one is when required.

Reply:
Yes, one is optional, one is required, and they run on disjoint sets of changes.